client/src/templates/layouts/NavigationLayout.js

```diff
@@ -1,20 +1,45 @@
 import { Drawer, List, Button, WhiteSpace } from "antd-mobile";
-import { Typography } from "antd";
-
-import React, { useState } from "react";
+import { Alert, Typography } from "antd";
+import axios from "axios";
+import React, { useState, useReducer } from "react";
 import { Link, useHistory } from "react-router-dom";
 import styled from "styled-components";
+
+import CookieAlert from "components/CookieAlert";
+import FeedbackSubmitButton from "components/Button/FeedbackModalButton";
+import Footnote from "components/Footnote";
 import { getInitials } from "utils/userInfo";
-import TextAvatar from "components/TextAvatar";
 import Header from "components/Header";
-import Footnote from "components/Footnote";
-import CookieAlert from "components/CookieAlert";
 import Main from "./Main";
 import MobileTabs from "./MobileTabs";
+import RadioGroup from "components/Feedback/RadioGroup";
+import RadioModal from "components/Feedback/RadioModal";
+import RatingModal from "components/Feedback/RatingModal";
+import FormInput from "components/Input/FormInput";
+import TextAvatar from "components/TextAvatar";
+import TextFeedbackModal from "components/Feedback/TextFeedbackModal";
+import ThanksModal from "components/Feedback/ThanksModal";
+import withLabel from "components/Input/with-label";
+import { ORANGE_RED, WHITE, ROYAL_BLUE, TROPICAL_BLUE } from "constants/colors";
```

Comment:
We shouldn't use this, as we are already setting the colors in the theme. See line 43